resources/README.md

```diff
@@ -1,5 +1,7 @@
 # Resources
 
+You need to run `go generate ./...` to download some resources manually.
```

Comment:
I am wondering if we can hint this in the test somehow

Reply:
maybe adding comments to `mysql.Install()`?

Reply:
I mean if we can make test print out the error and suggest to run go generate

Reply:
I haven't found any solution to define go compiler's error message.

But I think adding comments to var `resources` might be okay, since the compiler hints at the position of this var
![CleanShot 2022-03-07 at 16 28 42](https://user-images.githubusercontent.com/57934883/156995120-c61e1d90-3952-426e-9918-3a4e16e2a919.png)